pennylane_qiskit/qiskit_device2.py

```diff
@@ -536,8 +503,11 @@ def _process_estimator_job(measurements, job_result):
         along with some metadata. Extract the relevant number for each measurement process and
         return the requested results from the Estimator executions."""
 
-        expvals = job_result.values
-        variances = [res["variance"] for res in job_result.metadata]
+        expvals = [res.data.evs.item() for res in job_result]
+        variances = [
+            res.data.stds.item() ** 2 * 4096 for res in job_result
+        ]  # this 4096 is the # of shots
```

Comment:
A bit confused, the number of shots are hard-coded? Where does this `4096` value come from?

Reply:
Right now, the number of shots are hard-coded since this PR does not address how we process keyword arguments. Therefore, I've hard coded to # of shots here to be the default amount of shots that QIskit uses to pass the tests for now. In #547, this will be a variable either `circuit.shots.total_shots` or `dev.shots`.